tests/cli/commands/test_task_command.py

```diff
@@ -69,6 +72,15 @@ def move_back(old_path, new_path):
     shutil.move(new_path, old_path)
 
 
+@pytest.fixture()
+def freeze_time():
+    timestamp = "2022-06-10T12:02:44+00:00"
+    freezer = time_machine.travel(timestamp, tick=False)
+    freezer.start()
+    yield
+    freezer.stop()
```

Comment:
This fixture doesn't seem to be used anywhere?

Reply:
yup, removed